include/MFRC522_func.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * Helper routine to dump a byte array as hex values to Serial.
+ */
+void dump_byte_array(byte *buffer, byte bufferSize) {
+    for (byte i = 0; i < bufferSize; i++) {
+        Serial.print(buffer[i] < 0x10 ? " 0" : " ");
+        Serial.print(buffer[i], HEX);
+    }
+}
+
+bool authenticate(byte trailerBlock, MFRC522::MIFARE_Key key) {
```

Comment:
Better to `#include <MFRC522.h>`

[Include what you use](https://google.github.io/styleguide/cppguide.html#Include_What_You_Use)